adapters/brave/brave_test.go

```diff
@@ -0,0 +1,28 @@
+package brave
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderBrave, config.Adapter{
+		Endpoint: "http://point.braveglobal.tv/?t=3&partner={{.PublisherID}}"})
```

Comment:
Please consider using an obviously fake url for testing. There is no need for this to match the real endpoint and might be harder to maintain if you need to change it in the future.

Reply:
oh ok, got it , thanks